v3-sql-v4-sql/README.md

```diff
@@ -10,6 +10,7 @@
 
 - Nodejs v14/v16
 - Yarn
+  - we recommend v3.2.0
```

Comment:
Sorry but this is not the case, generally speaking we recommend yarn v1 not yarn v3 or yarn berry